### ISSUE TITLE
Add lxml to the list of requirements.

### ISSUE THREAD

**Issue by imansour12** [aditya-fpv/4chan-Downloader-Desu#1]
I tried to run this script on my Pop OS system and it failed, this was the error message:
`bs4.FeatureNotFound: Couldn't find a tree builder with the features you requested: lxml. Do you need to install a parser library?
`
Someone had asked about this on stackoverflow here: https://stackoverflow.com/questions/24398302/bs4-featurenotfound-couldnt-find-a-tree-builder-with-the-features-you-requeste
The way you solve it is you do `pip install lxml`
In conclusion, please add that to the list of dependencies in the README.md file.

**Comment by aditya-fpv** [comment]
Sorry for the late response.

I modified the script eliminating any need for external dependencies. If you have python  installed then you are good to go. Let me know if it doesn't work on a specific board

**Comment by imansour12** [comment]
It worked!!!🎉🎉
